--- a/Core/System.cpp	2025-10-16 09:26:40.102251400 +0200
+++ b/Core/System.cpp	2025-10-16 12:59:01.346842136 +0200
@@ -768,14 +768,8 @@
 }
 
 Path GetSysDirectory(PSPDirectories directoryType) {
-	const Path &memStickDirectory = g_Config.memStickDirectory;
-	Path pspDirectory;
-	if (!strcasecmp(memStickDirectory.GetFilename().c_str(), "PSP")) {
-		// Let's strip this off, to easily allow choosing a root directory named "PSP" on Android.
-		pspDirectory = memStickDirectory;
-	} else {
-		pspDirectory = memStickDirectory / "PSP";
-	}
+	Path memStickDirectory = Path("/storage/roms/savestates/PPSSPPSDL");
+	Path pspDirectory = Path("/storage/roms/savestates/PPSSPPSDL/PSP");
 
 	switch (directoryType) {
 	case DIRECTORY_PSP:
@@ -799,16 +793,13 @@
 	case DIRECTORY_SAVESTATE:
 		return pspDirectory / "PPSSPP_STATE";
 	case DIRECTORY_CACHE:
-		return pspDirectory / "SYSTEM/CACHE";
+		return Path("/storage/.config/ppsspp/PSP/SYSTEM/CACHE");
 	case DIRECTORY_TEXTURES:
 		return pspDirectory / "TEXTURES";
 	case DIRECTORY_PLUGINS:
 		return pspDirectory / "PLUGINS";
 	case DIRECTORY_APP_CACHE:
-		if (!g_Config.appCacheDirectory.empty()) {
-			return g_Config.appCacheDirectory;
-		}
-		return pspDirectory / "SYSTEM/CACHE";
+		return Path("/storage/.config/ppsspp/PSP/SYSTEM/CACHE");
 	case DIRECTORY_VIDEO:
 		return pspDirectory / "VIDEO";
 	case DIRECTORY_AUDIO:
@@ -819,11 +810,11 @@
 		return pspDirectory / "themes";
 
 	case DIRECTORY_MEMSTICK_ROOT:
-		return g_Config.memStickDirectory;
+		return memStickDirectory;
 	// Just return the memory stick root if we run into some sort of problem.
 	default:
 		ERROR_LOG(Log::FileSystem, "Unknown directory type.");
-		return g_Config.memStickDirectory;
+		return memStickDirectory;
 	}
 }
 
